package.json

```diff
@@ -42,50 +42,49 @@
     "@native-html/iframe-plugin": "^2.6.1",
     "@native-html/table-plugin": "^5.3.1",
     "@react-native-async-storage/async-storage": "1.23.1",
-    "@react-native-community/datetimepicker": "8.0.1",
-    "@react-native-community/netinfo": "11.3.1",
-    "@react-native-masked-view/masked-view": "0.3.1",
+    "@react-native-community/datetimepicker": "8.2.0",
+    "@react-native-community/netinfo": "11.4.1",
+    "@react-native-masked-view/masked-view": "0.3.2",
     "@react-navigation/bottom-tabs": "^6.5.9",
     "@react-navigation/drawer": "^6.6.4",
     "@react-navigation/elements": "^1.3.19",
     "@react-navigation/native": "^6.1.8",
     "@react-navigation/stack": "^6.3.18",
     "@reactvision/react-viro": "^2.41.6",
-    "@sentry/react-native": "~5.24.3",
-    "@shopify/flash-list": "1.6.4",
+    "@sentry/react-native": "~6.3.0",
+    "@shopify/flash-list": "1.7.1",
     "apollo-cache-inmemory": "1.6.6",
     "apollo-cache-persist": "~0.1.1",
     "apollo-client": "2.6.10",
     "apollo-link": "~1.2.12",
     "apollo-link-context": "1.0.20",
     "apollo-link-http": "1.5.17",
     "apollo-link-rest": "~0.7.3",
-    "expo": "~51.0.39",
-    "expo-application": "~5.9.1",
-    "expo-asset": "~10.0.10",
-    "expo-av": "~14.0.7",
-    "expo-barcode-scanner": "~13.0.1",
-    "expo-blur": "~13.0.2",
-    "expo-build-properties": "~0.12.5",
-    "expo-camera": "~15.0.16",
-    "expo-clipboard": "~6.0.3",
-    "expo-constants": "~16.0.2",
-    "expo-dev-client": "~4.0.29",
-    "expo-device": "~6.0.2",
-    "expo-document-picker": "~12.0.2",
-    "expo-font": "~12.0.10",
-    "expo-image-picker": "~15.0.7",
-    "expo-linear-gradient": "~13.0.2",
-    "expo-linking": "~6.3.1",
-    "expo-localization": "~15.0.3",
-    "expo-location": "~17.0.1",
-    "expo-media-library": "~16.0.5",
-    "expo-notifications": "~0.28.19",
-    "expo-screen-orientation": "~7.0.5",
-    "expo-secure-store": "~13.0.2",
-    "expo-splash-screen": "~0.27.7",
-    "expo-status-bar": "~1.12.1",
-    "expo-updates": "~0.25.27",
+    "expo": "^52.0.0",
+    "expo-application": "~6.0.2",
+    "expo-asset": "~11.0.2",
+    "expo-blur": "~14.0.2",
+    "expo-build-properties": "~0.13.2",
+    "expo-camera": "~16.0.11",
+    "expo-clipboard": "~7.0.1",
+    "expo-constants": "~17.0.4",
+    "expo-dev-client": "~5.0.9",
+    "expo-device": "~7.0.2",
+    "expo-document-picker": "~13.0.2",
+    "expo-font": "~13.0.3",
+    "expo-image-picker": "~16.0.4",
+    "expo-linear-gradient": "~14.0.2",
+    "expo-linking": "~7.0.4",
+    "expo-localization": "~16.0.1",
+    "expo-location": "~18.0.5",
+    "expo-media-library": "~17.0.5",
+    "expo-notifications": "~0.29.12",
```

Comment:
important for us?

> expo-notifications trigger types changed: we simplified calendar trigger input types in response to feedback that the old approach was difficult to use and error prone.

Reply:
I think this is only for local notifications.

Reply:
ok i found this PR...it is something with calendars we do not use https://github.com/expo/expo/pull/31598